content/user/advanced-modules/Cases with Portal.adoc

```diff
@@ -50,8 +57,9 @@ AOP settings through the admin panel.
 image:145AOP_settings.png[title="AOP Settings"]
 
 Here you can enable or disable the AOP functionality, configure the
-Joomla URL, set the case distribution method, “email from” details and
-the different email templates.
+Joomla URL,when configuring the Joomla URL be sure to use your local network IP instead of localhost or 127.0.0.1,
```

Comment:
this specific to networking with docker when working locally I assume and so may need to be more specific whats being refereed to here